workshop/1-Personalization/1.2-Real-Time-Events-Segment.ipynb

```diff
@@ -0,0 +1,451 @@
+    "\n",
+    "Next, you need to set the region for the Lambda that is deployed in your account, if you are running the workshop in us-west-2, then there is no need to change this setting, however for most workshops we use us-east-1.\n",
+    "\n",
+    "![Segment Destination Setup](images/segment/segment-step-22.png)\n",
+    "\n",
+    "\n",
+    "![Segment Destination Setup](images/segment/segment-step-23.png)\n",
+    "\n",
+    "Finally, click the slider button at the top of the screen to enable the destination.  You must enable the destination or events will not be sent to Personalize in the following steps.\n",
+    "\n",
+    "![Segment Destination Setup](images/segment/segment-step-24.png)"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "## Configure Lambda Parameters and Review Code\n",
+    "\n",
+    "Before the destination can send events to your Amazon Personalize events tracker, you will need to tell the destination lambda where to send the events.  It looks for an environment variable called 'personalize_tracking_id'.\n",
+    "\n",
+    "Let's set that.  Run the following cell to look up the relevant Amazon Personalize tracker from the Personalize workbook.\n",
+    "\n",
+    "We can then set the appropriate value in the destination Lambda."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ssm = boto3.client('ssm')\n",
+    "\n",
+    "# First, let's look up the appropriate tracking string\n",
+    "response = ssm.get_parameter(\n",
+    "    Name='retaildemostore-personalize-event-tracker-id'\n",
+    ")\n",
+    "\n",
+    "tracking_id = response['Parameter']['Value']\n",
+    "\n",
+    "print(tracking_id)"
+   ]
+  },
+  {
+   "source": [
+    "Go to your AWS console tab or window, and select Lambda from the Services menu.\n",
+    "\n",
+    "Find the SegmentPersonalizeEventsDestination, and click on it in the list.\n",
+    "\n",
+    "![Segment Events Destination Lambda](images/segment/segment-find-lambda-function.png)\n",
+    "\n",
+    "![Segment Events Destination Lambda](images/segment/segment-events-lambda.png)\n",
+    "\n",
+    "Then, scroll down to the parameters section.\n",
+    "\n",
+    "![Segment Events Destination Lambda](images/segment/segment-events-lambda-params.png)\n",
+    "\n",
+    "The tracking parameter should be set to the tracker from the first workbook.  \n",
+    "\n",
+    "Click the edit button, then paste in the tracking ID from the cell above, and click the Redeploy button at the top of the screen.\n",
+    "\n",
+    "Take some time to look at the code that this Lambda uses to send events to Personalize.  You can use this code in your own deployment, however you may need to change the event parameters sent to Amazon Personalize depending on the dataset you set up.\n",
+    "\n",
+    "```\n",
```

Comment:
Append `python` to get syntax highlighting of source (e.g. "```python").

Reply:
Oh excellent suggestion!  Will do that.